### PR TITLE
pjdfstests, fix old bep file

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: dev-util/pjdfstest/pjdfstest-0.1.recipe [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	cmd:autoconf
+	cmd:awk
+	cmd:gcc
+	cmd:make
+	"
+
+BUILD()
+{
+	autoreconf -ifs
+	LDFLAGS="-lnetwork" runConfigure ./configure
```
Comment (korli) [inline dev-util/pjdfstest/pjdfstest-0.1.recipe]: LDFLAGS can't be moved in the patch?

Comment (Begasus) [inline dev-util/pjdfstest/pjdfstest-0.1.recipe]: configure is created in the build, there is no mention of "socket, bind" or anything usefull atm in there ...

Comment (korli) [inline dev-util/pjdfstest/pjdfstest-0.1.recipe]: Ok, just add a 
AC_SEARCH_LIBS(socket, network) in configure.ac

**Comment by Begasus** [comment]
@korli ... updated, thanks!

**Pull request merged**